### ISSUE TITLE
Hyper-sensitive rule: `no-use-extend-native/no-use-extend-native`

### ISSUE THREAD

**Issue by lukeed** [eslint/eslint#7410]
The `no-use-extend-native/no-use-extend-native` rule is overly sensitive. In a `browser` setting, this makes sense for adding/modifying something like a `String.prototype`, but I'm seeing errors for wrapping functions with Bluebird. (see below)

Using XO ([XO config](https://github.com/sindresorhus/eslint-config-xo/blob/master/index.js)) and:

* ESLint: `1.10.3`
* Node: `5.4`, `6.6`, `6.8`
* Code `envs`: `node`

**What did you do? Please include the actual source code causing the issue.**

```js
// common:
const Promise = require('bluebird');
// 1.
const del = Promise.promisify(require('rimraf'));
// 2. 
return yield Promise.reduce(items, function () {...})
// 3.
const stat = Promise.promisify(fs.stat);
// 4.
const read = Promise.promisify(fs.readFile);
```

I'm aware that ESLint is unhappy because I've assigned Bluebird to the `Promise` name & thus am using methods that don't exist on the native `Promise`, but imo, ESLint should be smart enough to recognize that I've reassigned that keyword.

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report.

I think `no-use-extend-native/no-use-extend-native` is a custom rule, so it's not maintained by the ESLint team. I would recommend leaving an issue on [that rule's repo](https://github.com/dustinspecker/eslint-plugin-no-use-extend-native) instead.

**Comment by vitorbal** [comment]
Thanks for reaching out @lukeed!
@not-an-aardvark is right, as this plugin is not maintained by our team, your best bet would be to leave an issue on their repo.

**Comment by lukeed** [comment]
@not-an-aardvark @vitorbal Thank you both. I just blindly assumed that all rules were coming from ESLint directly --- whoops. 😜

**Comment by platinumazure** [comment]
@lukeed No worries. In future, any rule that is slash-delimited is a rule that comes from a plugin, and you can get the name of the plugin from the first part of the rule name. Hope this helps!

**Comment by samuelcastro** [comment]
@lukeed I'm experiencing the same issue and wondering about if have you opened an issue somewhere or have you fixed it?

Thanks.

**Comment by samuelcastro** [comment]
@lukeed Fixed it just overriding rules on package.json like:
```
"rules": {
  "no-use-extend-native/no-use-extend-native": 0
}
```

**Comment by lukeed** [comment]
@samuelcastro Yeah, I set it as a warning (`1`) instead.
